packages/yoastseo/src/scoring/assessments/inclusiveLanguage/helpers/notInclusiveWhenStandalone.js

```diff
@@ -2,6 +2,11 @@ import { isFollowedByException } from "./isFollowedByException";
 import { isFollowedByParticiple } from "./isFollowedByParticiple";
 import { nonNouns } from "../../../../languageProcessing/languages/en/config/functionWords";
 import { punctuationList } from "../../../../languageProcessing/helpers/sanitize/removePunctuation";
+import { getWords } from "../../../../languageProcessing";
+
+// Filter tokens from the exception lists that cause trouble when passed to getWords.
```

Comment:
Should we consider spelling out which trouble they cause? I think it's because they are multi-word punctuations/function words?